### DISCUSSION TITLE
00_pytorch_fundamentals - SyntaxError: invalid syntax

### DISCUSSION THREAD

**Discussion by diamondtusks** [mrdbourke/pytorch-deep-learning#144] [General]
im getting stuck on here for some reason, it says 'dtype:None' is an invalid syntax

float_32_tensor = torch.tensor([3.0, 6.0, 9.0],
                               dtype:None, 
                               device:None, 
                               requires_grad=False) 
float_32_tensor`

**Comment by diamondtusks** [comment]
All sorted, i rewrote it and it worked. Maybe something in my notations.

**Reply by mrdbourke** [reply]
Glad to hear you got it fixed!
